### DISCUSSION TITLE
关于资源库和备份盘切换连接的问题

### DISCUSSION THREAD

**Discussion by resultzh** [messense/aliyundrive-webdav#928] [Q&A] [answered]
设置好连接上默认是备份盘，怎么修改为资源库啊大佬

**Comment by onedaydayde** [comment, marked as answer]
配置里边，有设置，#备份盘：backup，资源盘：resource，旧模式：default，你的type对应那个就是用的那个
type=backup

**Reply by resultzh** [reply]
群晖docker版本里面没有找到配置选项大佬。。。怎么整求指导，谢谢啦

**Comment by resultzh** [comment]
搞定了，添加了一个环境变量DRIVE_TYPE为resource就可以了，谢谢大家
